build.gradle

```diff
@@ -317,76 +312,61 @@ repositories {
   }
 }
 
+configurations.configureEach {
+  resolutionStrategy.eachDependency { details ->
+    if (details.requested.name == 'commons-lang3') {
+      details.useVersion versions.commonsLang3
+    }
+  }
+}
+
 dependencies {
```

Comment:
please move to existing section below dependencies

Reply:
I've realised it is not needed at all

Reply:
we can remove empty config below then, if it is no longer needed